src/patterns/security-code/index.md.njk

```diff
@@ -0,0 +1,135 @@
+---
+title: Confirm a phone number
+description: Identifying users when they sign in
+section: Patterns
+theme: Help users to…
+aliases: 2FA, MFA, multi-factor authentication, security code, text message, two-factor authentication
+backlog_issue_id: 25
+layout: layout-pane.njk
+---
+
+{% from "_example.njk" import example %}
+
+Check that a user has access to a specific mobile phone number using a security code sent by text message.
+
+{{ example({group: "patterns", item: "security-code", example: "default", html: true, nunjucks: true, open: false}) }}
+
+## When to use this pattern
+
+Ask users to enter a security code when they need to sign in or complete a higher-risk task, such as changing a password.
+
+Asking users to enter a security code sent to their mobile phone gives a second layer of security over a [password](/patterns/passwords/).
+
+You can ask for a security code every time a user signs in or only once per device, depending upon the risk of your service.
+
+## How it works
+
+Send and ask the user for the security code when they: 
+* create an account
+* sign in later
+
+### When creating an account
+
+When the user creates an account, ask for their password and mobile phone number at the same time. Make it clear if you will only use the mobile number for sending security codes, or if you will use it for other purposes.
+
+After saving the user’s password and mobile phone number, verify their mobile phone number by sending them a text message with a 5 digit code in this format:
+
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{{ govukInsetText({
+  text: "12345 is your [service name] security code"
+}) }}
+
+Then ask the user to enter this code:
+
+{{ example({group: "patterns", item: "security-code", example: "default", html: true, nunjucks: true, open: false}) }}
+
+Ignore any formatting or punctuation such as spaces, hyphens and dashes.
+
+Give a time limit of 15 minutes for users to enter the code. If this time limit expires, display a ‘code has expired’ error message and send the user a new code.
+
+If the user follows the ‘Not received a text message?’ link, allow them to check which mobile number they entered, and to change it if necessary. This prevents the user becoming stuck if they entered a mobile number incorrectly. Do not allow users to change the number when signing in.
+
+{{ example({group: "patterns", item: "security-code", example: "resend-first-time", html: true, nunjucks: true, open: false}) }}
+
+This should send a new code, with its own 15 minute expiry window. The previous code should remain valid within its own expiry time.
```

Comment:
Tried to make this clearer and keep the "time limit" terminology used earlier.

```suggestion
This should send a new code, with its own 15 minute time limit. The previous code should remain valid within its original time limit.
```

Reply:
Good shout, thanks!